examples/apprunner/service.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apprunner.aws.upbound.io/v1beta1
+kind: Service
+metadata:
+  annotations:
+    meta.upbound.io/example-id: apprunner/v1beta1/service
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    region: us-east-1
+    serviceName: example
```

Comment:
Having `serviceName` as a parameter instead of using `metadata.name` does not look intuitive.

However, I believe there is no way to achieve this currently since the external name contains a randomly generated string by the provider. Crosslinking the related issue here: https://github.com/upbound/upjet/issues/142